espnet/nets/pytorch_backend/e2e_tts_transformer.py

```diff
@@ -597,6 +597,9 @@ def inference(self, x, inference_args, spemb=None, *args, **kwargs):
         threshold = inference_args.threshold
         minlenratio = inference_args.minlenratio
         maxlenratio = inference_args.maxlenratio
+        use_att_constraint = getattr(inference_args, "use_att_constraint", False)  # keep compatibility
+        if use_att_constraint:
+            logging.warn("Attention constraint is not yet supported in Transformer. Not enabled.")
```

Comment:
```suggestion
            logging.warning("Attention constraint is not yet supported in Transformer. Not enabled.")
```

warn is deprecated https://docs.python.org/3/library/logging.html#logging.Logger.warning

Reply:
Done.